terraform/groups/ecs-service/locals.tf

```diff
@@ -0,0 +1,92 @@
+# Define all hardcoded local variable and local variables looked up from data resources
+locals {
+  stack_name                = "filing-create" # this must match the stack name the service deploys into
```

Comment:
Think this might need changing to `company-requests-service`.

Reply:
Have updated this . Wondering if this shouldn't be a variable.

Reply:
Change looks good. Why do you think it should be a variable?

Reply:
The value varies just like the field below, so like that it could be a variable.

Reply:
I think it's fine how we've defined it - just as we did for OE Web but a different value. The local definition just below uses `var.environment` but that is then reused several times. `company-requests-service` seems to be used (and therefore defined) just once.